docs/plugins/development/index.md

```diff
@@ -138,29 +138,46 @@ Any additional apps must be installed within the same Python environment as NetB
 
 ## Create setup.py
```

Comment:
Looks like there are still some latent references to `setup.py` here, above in the discussion of the file structure, and below in the development installation section.